update-dois/src/test/java/org/reactome/release/updateDOIs/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package org.reactome.release.updateDOIs;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
This would be a good time to learn about writing unit tests with JUnit! There are some good, recent examples in the release-qa project. Also, this class should probably be renamed to something like "UpdateDOITest".